text/0000-icons.md

```diff
@@ -65,19 +68,43 @@ A variable containing an inline SVG element that can be placed directly into you
 
 >**This is the bulk of the RFC. Explain the design in enough detail for somebody familiar with a TVA library to understand, and for somebody familiar with the implementation to implement. This should get into specifics and corner-cases, and include examples of how the feature is used. Any new terminology should be defined here.**
 
-SVG icons provided by designers are programmatically optimized and the syntax is modified based on the use case (such as removing `xmlns` attributes for inline).
+SVG icons provided by designers are placed in the `src/svg` folder, then programmatically optimized using `svgo` and saved to `build/svg`.
```

Comment:
I'm not sure we need an `svg` folder since by default that's the foundation of the entire workspace?

Reply:
I had two reasons for this.  1) Not overwriting the original files with the optimized ones.  And 2) Making the SVG files available for anyone not using one of our supported frameworks.